.pfnci/linux/tests/cuda-rapids.Dockerfile

```
@@ -1,5 +1,4 @@
-# TODO(kmaehashi): Fix stable image after release (#6805)
-ARG BASE_IMAGE="rapidsai/rapidsai-core-dev-nightly:22.08-cuda11.5-devel-ubuntu20.04-py3.8"
+ARG BASE_IMAGE="rapidsai/rapidsai-core-dev:23.06-cuda11.8-devel-ubuntu22.04-py3.10"
```

Thanks! Let's pin to the old one and fix that in a separate pull-request.

```suggestion
ARG BASE_IMAGE="rapidsai/rapidsai-core-dev:22.10-cuda11.5-devel-ubuntu20.04-py3.9"
```